### DISCUSSION TITLE
Not anything major, do we have a community discord?

### DISCUSSION THREAD

**Discussion by JacobHin2** [leftwm/leftwm#452] [Ideas]
It seems important, because it makes it easy to troubleshoot user problems faster, or even not at all because you could have FAQ channels. 

This would also free up some of the opened issues because we could ask the developers directly.

Please consider this and let me know. I don't think a discussion is apt, maybe I should've opened an issue so apologies in advance if this is the wrong place :)

**Comment by mautamu** [comment]
I have created one: https://discord.gg/c9rB3wDnVs

Tagging @lex148 and @VuiMuich, would be more than happy to give one of y'all ownership of the server as necessary.

Best,
Mautamu

**Reply by VuiMuich** [reply]
I'd definitely be ok with being 'co-owner', if ownership is only possible to be assigned to a single user I think it should be held by @lex148
